unitTests/ui/DynamicFacetManagerTest.ts

```diff
@@ -28,6 +28,13 @@ export function DynamicFacetManagerTest() {
         DynamicFacetTestUtils.createAdvancedFakeFacet({ id: 'test2', field: '@test2', numberOfValues: 5 }).cmp,
```

Comment:
Are the tests for the new functionality going to be included in a separate PR?

Reply:
I'll do it soon on this same PR